site/book/05-developing-functions/03-developing-in-Typescript.md

```diff
@@ -219,9 +215,7 @@ Currently supported platforms: amd64 Linux/Mac
 1. Run `expand-team-cr` on `example-configs`.
 
    ```shell
-   $ kpt fn source $CONFIGS \
-    | node dist/expand_team_cr_run.js \
-    | kpt fn sink $CONFIGS
+   $ kpt fn eval $CONFIGS --exec "node dist/expand_team_cr_run.js"
```

Comment:
Assuming you manually tested this flow with node?

Reply:
yes